MAUI/TreeMap/getting-started.md

```diff
@@ -0,0 +1,515 @@
+
+## Creating an Application using the .NET MAUI TreeMap
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the SfTreeMap to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.TreeMap`, then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.TreeMap` in XAML or C# code.
+
+4. Initialize `SfTreeMap`.
+
+
+{% tabs %}
+{% highlight XAML hl_lines="3 4" %}
+
+<ContentPage
+...        
+xmlns:treemap="clr-namespace:Syncfusion.Maui.TreeMap;assembly=Syncfusion.Maui.TreeMap">
+    <treemap:SfTreeMap />
+</ContentPage>
+
+{% highlight C# hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.TreeMap;
+. . .
+
+public partial class MainPage : ContentPage
+{
+public MainPage()
+{
+InitializeComponent();
+SfTreeMap treeMap = new SfTreeMap();
+this.Content = treeMap;
+}
+}
+
+{% endhighlight %}
+
+## Register the handler
+
+The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.html) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 8" %}
+
+    
+    using Syncfusion.Maui.Core.Hosting;
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Populating treemap items
+
+Here are the steps to render treemap items using the .NET MAUI TreeMap control with respective data properties created in a data model class.
+
+* Create a data model for treemap.
+* Create view model.
+* Bind data source for TreeMap.
+
+#### Create a data model for treemap 
+
+Create a simple data model in a new class file as shown in the following example code.
+
+{% tabs %}
+{% highlight C# tabtitle="PopulationDetails.cs" %}
+
+// Represents demographic information for a country.
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Create view model
+
+Create a view model class to set values for the properties listed in the model class as shown in the following example code.
+
+{% tabs %}
+{% highlight C# tabtitle="PopulationViewModel.cs" %}
+
+// Presents demographic information for countries in the Americas for the year 2023.
```

Comment:
Represents demographic information for countries in the Americas for the year 2023.

Reply:
Content changed